Steamfitter.Api/Services/TaskExecutionService.cs

```diff
@@ -537,6 +549,66 @@ private STT.Task<string> RunTask(TaskEntity taskToExecute, ResultEntity resultEn
             return task;
         }
 
+        private async STT.Task<string> HttpTaskTask(TaskEntity taskToExecute)
+        {
+            HttpResponseMessage response;
+            using (var scope = _scopeFactory.CreateScope())
+            {
+                var tokenResponse = await ApiClientsExtensions.GetToken(scope);
```

Comment:
We should make this re-use tokens in the future.

Reply:
added a TODO